src/Frontend/Topics.hs

```diff
@@ -0,0 +1,137 @@
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE FlexibleInstances   #-}
+{-# LANGUAGE KindSignatures      #-}
+{-# LANGUAGE LambdaCase          #-}
+{-# LANGUAGE OverloadedStrings   #-}
+
+{-# OPTIONS_GHC -Werror -Wall #-}
+
+module Frontend.Topics
```

Comment:
What if we move it to `Frontend.Page.Topics` or should I move `Frontend.Page.CreateIdea` to `Frontend.CreateIdea`? I prefer collect every page under the Page submodule.